csc302-spring-2021/proj-DaTeam
# Frontend form

## Description



- Fixed frontend and shared fail build issue

## Checklist


## QA Steps



1. ```cd backend && npm run dev && cd ../frontend  && npm start   ``` and make sure everything is running
